index.html

```diff
@@ -52,7 +52,271 @@ <h3>
+                Ruby
+              </li>
+            </ul>
+            <button type="submit" class="see-project-button-full">
+              See Project
+            </button>
+          </li>
+
+          <li class="recent-works">
+            <h3>
+              Profesional Art Printing Data
+            </h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <ul class="technologies-used-recent-works">
+              <li>
+                html 
+              </li>
+              <li>
+                bootstrap
+              </li>
+              <li>
+                Ruby
+              </li>
+            </ul>
+            <button type="submit" class="see-project-button-full">
+              See Project
+            </button>
+          </li>
+
+          <li class="recent-works">
+            <h3>
+              Profesional Art Printing Data
+            </h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <ul class="technologies-used-recent-works">
+              <li>
+                html 
+              </li>
+              <li>
+                bootstrap
+              </li>
+              <li>
+                Ruby
+              </li>
+            </ul>
+            <button type="submit" class="see-project-button-full">
+              See Project
+            </button>
+          </li>
+
+          <li class="recent-works">
+            <h3>
+              Profesional Art Printing Data
+            </h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <ul class="technologies-used-recent-works">
+              <li>
+                html 
+              </li>
+              <li>
+                bootstrap
+              </li>
+              <li>
+                Ruby
+              </li>
+            </ul>
+            <button type="submit" class="see-project-button-full">
+              See Project
+            </button>
+          </li>
+
+          <li class="recent-works">
+            <h3>
+              Profesional Art Printing Data
+            </h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+            </p>
+            <ul class="technologies-used-recent-works">
+              <li>
+                html 
+              </li>
+              <li>
+                bootstrap
+              </li>
+              <li>
+                Ruby
+              </li>
+            </ul>
+            <button type="submit" class="see-project-button-full">
+              See Project
+            </button>
+          </li>
+        </ul>
+      </section>
+
+
+
+      <section class="about-me-container">
```

Comment:
Your about section also does not match the Figma template of your choosing. Consider fixing it.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/178111676-ed3b2be6-2ffa-4a72-8bcb-19fe69905ae9.png) | ![image](https://user-images.githubusercontent.com/33268927/178111703-f622aa1f-1bfb-4094-9dcf-f5dd736eda46.png) |